test/unit/schema/postgres.js

```diff
@@ -786,6 +786,19 @@ describe("PostgreSQL SchemaBuilder", function() {
       expect(spy.secondCall.args).to.deep.equal(['email', 'email context']);
       expect(spy.thirdCall.args).to.deep.equal(['users', 'table context']);
     });
+
+    it('TableCompiler calls wrapIdentifier when altering column', function () {
+      client
```

Comment:
How does this test test that wrapIdentifier was called correctly?

Reply:
ah right, that spy was set on wrapIdentifier function...  original naming of that spy should probably have been done better :) 👍 This is good.